src/target/riscv_debug.h

```diff
@@ -204,6 +204,8 @@ typedef struct riscv_hart {
 // so they cannot collide with GPR registers, so you have to subtract
 // RV_CSR_GDB_OFFSET from the value received from gdb
 #define RV_CSR_GDB_OFFSET 128
+#define RV_FPU_GDB_FIRST_REGISTER 33
```

Comment:
The original idea with this section of defines is that it was a block just for the CSRs - perhaps move these two new defines below to line 218 as a new block just for FPU definitions?